src/tools/gophish_export.py

```diff
@@ -253,6 +253,86 @@ def get_application(rawEvent):
     return application
 
 
+def find_unique_target_clicks_count(clicks):
+    """Return the number of unique clicks in a click dict."""
+    uniq_list = list()
+    for click in clicks:
+        if click["user"] not in uniq_list:
+            uniq_list.append(click["user"])
+    return len(uniq_list)
```

Comment:
I would just use a set here instead of list, then you don't need the `if` statement:
```suggestion
    unique_users = set()
    for click in clicks:
        unique_users.add(click["user"])
    return len(unique_users)
```

Reply:
See 5816a24f37e9a600f2b55946de18b7acb8d0da07 for revisions